code/modules/mob/living/carbon/human/life/handle_disabilities.dm

```diff
@@ -79,10 +79,17 @@
 						"why woud i take a pin pointner??", \
 						"FUCK IT; KISSYOUR ASSES GOOD BYE DEAD MEN! I AM SELFDESTRUCKTING THE STATION!!!!", \
 						"How do I set up the. SHow do I set u p the Singu. how I the scrungularity????", \
-						"OMG I SED LAW 2 U FAG MOMIM LAW 2!!!"))
+						"OMG I SED LAW 2 U FAG MOMIM LAW 2!!!", \
+						"Latex vox are cute!", \
+						"HOG", \
+						";ITS LOOSE", \
```

Comment:
No shuttlecall baiting for braindamage lines, please

Reply:
ehhh, id be ok with removing the ;